website/docs/developer/research/research-graphql.md

```diff
@@ -237,6 +286,18 @@ The following tools might be useful for developers while working with GraphQL
 - GraphQL Boilerplates- useful for backend projects.
 - Apollo Graph Manager-  useful for monitoring the performance and usage of GraphQL backend.
 
+### Recommendation
+While GraphQL is an interesting technology, the lack of documentation makes implementation difficult.
+In particular, deployment and integration lack vital instructions, with most tutorials running in a docker container or Python script rather than a realistic environment.
+Theoretically GraphQL could be run in an EC2 instance with some form of connecting layer such as [Fargate](https://aws.amazon.com/fargate/?whats-new-cards.sort-by=item.additionalFields.postDateTime&whats-new-cards.sort-order=desc&fargate-blogs.sort-by=item.additionalFields.createdDate&fargate-blogs.sort-order=desc) but this is untested and would need research and cost analysis.
+
+Should this be desired, I feel strongly that Graphene is the only known option that gives us the flexibility that is required to justify this level of adaptation.
```

Comment:
So the question I have, is moving to this technology where we have a single GraphQL server that interacts with the database a better solution than having several microservices (lambdas) that each have database code that interact with the database?

Do you feel that it would be
- more manageable/maintainable long term?
- faster to change/code against for features?
- provide us better flexibility?
- consolidate the amount of code we would need to maintain?
- provide other additional benefits over our current architecture?

Reply:
Bit difficult to answer given that so little worked. Did what I could.